slu/slu/utils/config.py

```diff
@@ -92,7 +142,8 @@ def __attrs_post_init__(self) -> None:
             const.CLASSIFICATION
         ][const.FORMAT]
 
-        self.rules = self._config[const.RULES]
+        # self.rules = self._config[const.RULES]
+        self.rules = self._config["postprocess"][0]["params"]["rules"]
```

Comment:
We can't guarantee that rules will always be at that position (always question when accessing list indices), it is better to handle this in a separate PR.

Reply:
yeah yeah. but that's what it took to run and pass the tests with future axis config.

Reply:
fixed it by reverting  to the old way itself, but if we don't fix it. none of the new `Config` would work. nor won't the tests.